core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+**[R7]**	The representatives and their authorized delegates who can perform commercial document authorizations SHOULD be explicitly listed or inferred from the stated legal delegation rules of the counterparties in the contract underlying any commercial document. 
+
+**[R8]**	A commercial document MUST be non-repudiable.
+
+Note that while non-repudiation in the physical world is most often tied to a physical signature of an individual on a legal document, in the digital world a digital signature over a digital legal document such as an Order or an Invoice belonging to a known and verifiable digital identity of a counter party serves the same purpose.
+
+Example
+A Buyer and Seller may agree that a signed Order requires a signed original paper copy, or a digitally signed electronic Order Form, in addition to an Order being a digitally signed and recorded within a Baseline Protocol Implementation.
+
+## 2.5 Distributed Ledger Technology (DLT)
+
+A distributed ledger (also called a shared ledger or distributed ledger technology or DLT) is a consensus of replicated, shared, and synchronized digital data spread across multiple sites. There is no central administrator or centralized data storage. 
+
+A DLT is the foundational enabler of a BPI with no or limited trust assumptions.
+
+A peer-to-peer network is required as well as a consensus algorithm to ensure replication across nodes is undertaken [put reference here]
+
+For specificity, the popular word "Blockchain" is a particular form of DLT design.
+
+## 2.6 Baseline Protocol Instance
+
+Baseline Protocol Instances (BPIs) are logical constructs shared between (commercial) counterparties of Requesters and Providers and implemented on a Distributed Ledger. They are used to either validate, or reconcile commercial transactions between Requesters and Providers related to all (commercial) state objects transacted between them. The nature of bi- or multi-lateral transactions is such that two or more parties may (commercially) transact to/from each other interchangeably. 
+
+Abstractly, a BPI consists of
+* private messaging between agreement counterparties about the state or the requested or finalized state changes of the agreement between them,
+* the representation of a (commercial) agreement and its business rules and data as distinct workflows and worksteps between its counterparties organized into workgroups based on the stipulations of the (commercial) agreement
+* the deterministic processing and finalization of state change requests based on commercial documents between the agreement counterparties as stipulated by the (commercial) agreement.
```

Comment:
should we capitalise 'Agreement Counterparties' too?

Reply:
yes ... done